config.default.yaml

```
@@ -19,7 +19,7 @@ countries: ["NG", "BJ"]
 # Can be replaced by country ["NG", "BJ"], continent ["Africa"] or user-specific region, see more at https://pypsa-earth.readthedocs.io/en/latest/configuration.html#top-level-configuration
 
 enable:
-  retrieve_databundle: true #  Recommended 'true', for the first run. Otherwise data might be missing.
+  retrieve_databundle: false #  Recommended 'true', for the first run. Otherwise data might be missing.
```

Apologies Virio, I've just seen this change, this likely leads to issues the first time the model is run.
Is it a refuse or is it intentional?

---

Its unintentional, reverting it now
```suggestion
  retrieve_databundle: true #  Recommended 'true', for the first run. Otherwise data might be missing.
```